examples/docker-compose.yml

```diff
@@ -18,3 +18,15 @@ services:
       - 6877:6877
       - 6878:6878
     healthcheck: {test: curl -f localhost:6878/api/readyz, interval: 1s, start_period: 35s}
+
+  frontegg:
```

Comment:
All the mocks and additional services for the docker compose are from [the](https://github.com/MaterializeInc/terraform-provider-materialize/pull/351). There is probably a better way to mirror this.